src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__block  calendar__block--start_sun"></div>
```

Comment:
All modifiers for calendar block should be written near calendar.
And make sure to follow BEM naming convention for complex modifiers: block-name--modifier-name--modifier-value (checklist)
```suggestion
    <div class="calendar calendar--month-length--31 calendar--start-day--sun">
      <div class="calendar__block"></div>
```